bnd.bnd

```diff
@@ -0,0 +1,10 @@
+Bundle-Description: HiveMQ Java MQTT Client 
+Bundle-License: Apache 2.0 License
+Export-Package: \
+  com.hivemq.client.mqtt.*, \
+  com.hivemq.client.mqtt.rx.*, \
+  com.hivemq.client.mqtt.rx.reactivestreams.*, \
```

Comment:
I am not a OSGI expert, but isn't `com.hivemq.client.rx.reactivestreams.*` covered by `com.hivemq.client.rx.*`?